### ISSUE TITLE
Note anchored Line Skipped by Alt Right Arrow

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#32294]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

You cannot navigate to a note anchored line using Alt Right arrows, but can if you move backwards.

STEPS:
1.  on the attached file navigate along the first few notes with right arrows.
Notice as expected navigation moves from note to note and speech announces the start and the end of the note anchored line attached to the note.
2.  Go back to the first note and this time hold down Alt while navigating along.  Notice that it skips over the edit panel for the note anchored line. 
3. Now use Alt left arrow to move backwards over the same notes.  Notice this time the navigation does move to the note anchored line itself.

This is similar to 31449 in that sometimes those mid note elements can be navigated to with Alt left arrow but are skipped with Alt right.

### Supporting files, videos and screenshots

[Note anchored Lines.zip](https://github.com/user-attachments/files/25433128/Note.anchored.Lines.zip)

### In which versions of MuseScore Studio is this issue present?

4.6

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
